axelor-account/src/main/resources/views/MoveTemplateLine.xml

```diff
@@ -4,7 +4,8 @@
   xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.4.xsd">
 
   <grid name="move-template-line-in-amount-grid" title="Move template line"
-    model="com.axelor.apps.account.db.MoveTemplateLine">
+    model="com.axelor.apps.account.db.MoveTemplateLine"
+    onNew="action-attrs-move-template-line-set-required-partner">
```

Comment:
we need to talk about this feature with Geoffrey, I'm not sure we really want to do this in AOS. 
Also I don't understand why in "move-template-line-in-amount-form" there is a partner but not in "move-template-line-in-amount-grid".
If you call "action-attrs-move-template-line-set-required-partner" on the "move-template-line-in-amount-grid" onNew, I'm not sure how the view will behave

Reply:
Added a verification on action-attrs to verify parent.Model